### PR TITLE
Move the gw configuration from the start.sh to the instance.env

### PR DESCRIPTION
**Pull request by taban03**
Signed-off-by: at670475 <andrea.tabone@broadcom.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [ ] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes 

#### Changes proposed in this PR

- 

#### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

File: playbooks/roles/configfmid/defaults/main.yml [hunk 1]
```diff
@@ -25,7 +25,13 @@ zowe_apiml_catalog_port: 7552
 zowe_apiml_discovery_port: 7553
 zowe_apiml_gateway_port: 7554
 zowe_apiml_verify_certficates_of_services: true
+# APIML configuration properties
 zowe_apiml_enable_sso: false
+zowe_apiml_allow_encoded_slashes: true
+zowe_apiml_prefer_ip_address: true
+zowe_apiml_gateway_timeout_millis: 600000
+zowe_apiml_security_x509_enabled: false
+zowe_apiml_security_zosmf_applid: IZUDFLT
```
Comment (stevenhorsman) [inline playbooks/roles/configfmid/defaults/main.yml:34]: we don't want the new properties in the fmid as that is based on 1.9

File: playbooks/roles/configfmid/tasks/configure_instance.yml [hunk 2]
```diff
@@ -30,6 +30,11 @@
     sed -e "s+^DISCOVERY_PORT=.*\$+DISCOVERY_PORT={{ zowe_apiml_discovery_port }}+" | \
     sed -e "s+^GATEWAY_PORT=.*\$+GATEWAY_PORT={{ zowe_apiml_gateway_port }}+" | \
     sed -e "s+^APIML_ENABLE_SSO=.*\$+APIML_ENABLE_SSO={{ zowe_apiml_enable_sso|string|lower }}+" | \
+    sed -e "s+^APIML_ALLOW_ENCODED_SLASHES=.*\$+APIML_ALLOW_ENCODED_SLASHES={{ zowe_apiml_allow_encoded_slashes|string|lower }}+" | \
+    sed -e "s+^APIML_PREFER_IP_ADDRESS=.*\$+APIML_PREFER_IP_ADDRESS={{ zowe_apiml_prefer_ip_address|string|lower }}+" | \
+    sed -e "s+^APIML_GATEWAY_TIMEOUT_MILLIS=.*\$+APIML_GATEWAY_TIMEOUT_MILLIS={{ zowe_apiml_gateway_timeout_millis }}+" | \
+    sed -e "s+^APIML_SECURITY_X509_ENABLED=.*\$+APIML_SECURITY_X509_ENABLED={{ zowe_apiml_security_x509_enabled|string|lower }}+" | \
+    sed -e "s+^APIML_SECURITY_ZOSMF_APPLID=.*\$+APIML_SECURITY_ZOSMF_APPLID={{ zowe_apiml_security_zosmf_applid|string|lower }}+" | \
```
Comment (stevenhorsman) [inline playbooks/roles/configfmid/tasks/configure_instance.yml:37]: again, not needed on the configurefmid

File: playbooks/roles/configfmid/tasks/main.yml [hunk 3]
```diff
@@ -28,6 +28,11 @@
     - zowe_apiml_discovery_port
     - zowe_apiml_gateway_port
     - zowe_apiml_enable_sso
+    - zowe_apiml_allow_encoded_slashes
+    - zowe_apiml_prefer_ip_address
+    - zowe_apiml_gateway_timeout_millis
+    - zowe_apiml_security_x509_enabled
+    - zowe_apiml_security_zosmf_applid
```
Comment (stevenhorsman) [inline playbooks/roles/configfmid/tasks/main.yml:35]: Same comment

**Review by stevenhorsman — CHANGES_REQUESTED** [review]

File: playbooks/roles/configfmid/tasks/configure_instance.yml (hunk 2 above)
Comment (taban03) [reply playbooks/roles/configfmid/tasks/configure_instance.yml:37]: Thank you Steven, I'm gonna fix it

**Comment by stevenhorsman** [comment]
convenience tests: https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-1741/9/
ptf tests: https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-1741/10/

**Review by stevenhorsman — APPROVED** [review]

**Pull request merged**